x11-wm/evilwm/evilwm-1.0.1-r1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils toolchain-funcs multilib
```

Comment:
Do you really need multilib eclass?

Reply:
also switch to `desktop.eclass` instead of `eutils`

Reply:
eutils replaced with desktop

Reply:
And multilib removed